.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish to PyPi
+
+on: 
+  release:
+    types: [created]
+
+jobs:
+  build-and-publish:
+    name: Build python package and publish to PyPi
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
```

Comment:
I think the docker containers in the repo are set up to use the latest python 3 (3.9 now I think).  It should not matter much but might be good to be consistent with the version the tests are run against.